packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -3637,16 +3637,6 @@ Map {
       },
     },
   },
-  "Form" => Object {
-    "propTypes": Object {
-      "children": Object {
-        "type": "node",
-      },
-      "className": Object {
-        "type": "string",
-      },
-    },
-  },
```

Comment:
This should be removing from this snapshot

Reply:
Should be fixed!